DigitalLearningSolutions.Web.Tests/Controllers/SuperAdmin/FaqControllerTests.cs

```diff
@@ -0,0 +1,59 @@
+namespace DigitalLearningSolutions.Web.Tests.Controllers.SuperAdmin
+{
+    using System;
+    using System.Collections.Generic;
+    using System.Linq;
+    using DigitalLearningSolutions.Data.Models.Support;
+    using DigitalLearningSolutions.Data.Services;
+    using DigitalLearningSolutions.Web.Controllers.SuperAdmin;
+    using DigitalLearningSolutions.Web.ViewModels.SuperAdmin.Faqs;
+    using FakeItEasy;
+    using FizzWare.NBuilder;
+    using FluentAssertions;
+    using FluentAssertions.AspNetCore.Mvc;
+    using NUnit.Framework;
+
+    public class FaqControllerTests
```

Comment:
This file should be CRLF line endings.

Reply:
Odd, it could be getting set git when committing/pushing, but if you've just opened this file, without modification, and it reads it as CRLF, then it looks like there is a discrepancy between how our IDEs are looking at it. If I open this, it is specifically LF (I'm using Visual Studio for reference). Running the formatter, it sets it to CRLF, so it looks like all the lines have changed.

Are you able to have a look in a different editor? Something simple like Notepad++ should be able to open the files and show line endings easily enough. I suspect something has gone wrong in your editor somewhere, and if Notepad++ shows them as CRLF, then we can confirm that.

Reply:
Ok thanks, Notepadd++ and rubymine are both showing CRLF

Reply:
Well colour me confused. If you have git auto-CRLF set to false, then I have no idea where this could be coming from, or why our editors are showing discrepancies. 

They definitely seem to have changed, as ViewModels/Support/Faqs/SearchableFaqViewModel.cs has the entire file highlighted as changed in the git differences, despite just the imports changing. On my end, it looks like the line endings have changed from before and after. Out of curiosity, what do your editors show the line endings are for that file, specifically the version currently on master?

Reply:
Very weird, I'm getting CRLF for everything locally including support/faqs/searchableFaqViewModel.cs on master

Reply:
I would expect pretty much everything on master to be CRLF as that is what we have tried to go for as a convention, though there are some places where LF has slipped through. But the fact that you get CRLF before, then it seems to want to change it to what it also sees as CRLF is extremely odd. I'll post a message in hee-internal to see if anyone has any ideas.